src/index.js

```diff
@@ -502,7 +475,7 @@ function createConfig(options, entry, format, writeMeta) {
 						extract:
 							!!writeMeta &&
 							options.css !== 'inline' &&
-							options.output.replace(EXTENSION, '.css'),
+							resolve(outputDir, `${entryFileName}.css`),
```

Comment:
I'd rather we not use `outputDir` as it isn't guaranteed to match up with expectations in the future (see #896)

CSS should continue to be output according to the result of [`getOutput`](https://github.com/developit/microbundle/blob/9a4e2b2096d3824661738edb142b4658cf3d9d0b/src/index.js#L231-L237) for resilience IMO.

Reply:
How about changing it to [this](https://github.com/developit/microbundle/pull/961)